src/lib/atoms/Badge/Badge.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { shallow, ShallowWrapper, mount } from 'enzyme';
+import Badge, { IBadgeProps } from './index';
+
+const shallowComponent = (props?: IBadgeProps): ShallowWrapper => {
```

Comment:
Please make one function out of these functions and initialize it in beforeEach.